rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -128,6 +128,22 @@
             @QueryParam("index") @DefaultValue("-1") int index, @QueryParam("limit") @DefaultValue("-1") int limit)
             throws PermissionRestException, NotConnectedRestException;
 
+    /**
+     * Returns a list of jobs with each jobs is described using -
+     * its id - its owner - the JobInfo class
+     *
+     * @param jobsId
+     *            the list of id of the jobs to return
+     * @param sessionId
+     *            a valid session id
+     * @return a list of UserJobData
+     */
+    @GET
+    @Path("listjobinfo")
+    @Produces({ "application/json", "application/xml" })
+    List<UserJobData> listJobInfo(@HeaderParam("sessionid") String sessionId, @QueryParam("jobsid") List<String> jobsId)
```

Comment:
Why not using a Set of jobsId?

Reply:
We use List to return the UserJobData in the same order we get the job IDs